source/_patterns/30-organisms/collection/collection.scss

```diff
@@ -25,12 +25,79 @@ $background-overlap-wide: 50px;
     background-color: transparent;
     box-shadow: none;
   }
+
+  &:first-child {
+    &.collection--branded {
+      .collection__branding-wrapper {
+        padding-top: 16px;
+
+        @include tablet-up {
+          padding-top: 32px;
+        }
+
+        @include desktop-wide {
+          padding-top: 40px;
+        }
+      }
+    }
+
+    &.collection--borderless {
+      .collection__branding-wrapper {
+        padding-top: 0;
+      }
+    }
+  }
+
+  &:last-child {
+    margin-bottom: 16px;
+
+    &.collection--branded {
+      margin-bottom: 0;
+
+      .collection__branding-wrapper {
+        padding-bottom: 16px;
+      }
+    }
+
+    @include tablet-up {
+      margin-bottom: 32px;
+
+      &.collection--branded {
+        margin-bottom: 0;
```

Comment:
ist diese zeile obsolet, weil in zeile 55 bereits definiert oder braucht's das wegen der spezifizität?

Reply:
-> Spezifizität mit MQ

Reply:
![image](https://user-images.githubusercontent.com/909895/52792147-f2390300-306a-11e9-9f53-dd9f85b53b4c.png)

Reply:
![image](https://user-images.githubusercontent.com/909895/52792246-30362700-306b-11e9-9eb8-513a69a2818f.png)